apps/dashboard/app/(app)/@breadcrumb/tags.ts

```diff
@@ -0,0 +1,7 @@
+
+export const tags = {
+    api: (apiId: string): string => `api-${apiId}`,
```

Comment:
these are not only relevant to breakdcrumbs let's move them to /app/lib/cache/tags.ts and also change all of the imports to `@/lib/cache`